salt/metalk8s/orchestrate/bootstrap_step1.sls

```
@@ -0,0 +1,46 @@
+Run bootstrap prechecks:
```

There is no deps between all these steps?

---

That's mostly package installation, configuration file creation, and cheks...
But maybe there's a missing "fail hard" when this 1st stage fails.